indico/cli/i18n.py

```diff
@@ -335,6 +335,68 @@ def remove_empty_pot_files(path: Path, python=False, javascript=False, react=Fal
         _get_messages_react_pot(path).unlink()
 
 
+# Merge after generating all the .pot files
+def merge_pot_files(output_file, *input_files):
+    """Merge multiple POT files into a single POT file."""
+    merged_catalog = None
+
+    print('\n🗨  Merging POT files...\n')
+
+    for input_file in input_files:
+        with input_file.open('rb') as f:
+            catalog = read_po(f)
+
+            if merged_catalog is None:
+                merged_catalog = catalog
```

Comment:
I don't really like this. Can't we just start with an empty catalog and then add data from all input files? It's much less obscure IMHO.